plugins/woocommerce/src/Utilities/OrderUtil.php

```diff
@@ -0,0 +1,80 @@
+<?php
+/**
+ * A class of utilities for dealing with orders.
+ */
+
+namespace Automattic\WooCommerce\Utilities;
+
+use Automattic\WooCommerce\Internal\DataStores\Orders\CustomOrdersTableController;
+use WC_Order;
+use WP_Post;
+
+/**
+ * A class of utilities for dealing with orders.
+ */
+final class OrderUtil {
+
+	/**
+	 * Helper function to get screen name of orders page in wp-admin.
+	 *
+	 * @return string
+	 */
+	public static function get_order_admin_screen() : string {
+		return self::is_cot_enabled()
+			? wc_get_page_screen_id( 'shop-order' )
+			: 'shop_order';
+	}
+
+	/**
+	 * Helper function to get whether custom order tables are enabled or not.
+	 *
+	 * @return bool
+	 */
+	public static function is_cot_enabled() : bool {
```

Comment:
I wonder if this method still makes sense once the class becomes non-static, as the usage would be:

```php
wc_get_container()->get( OrderUtil::class )->is_cot_enabled();
```

which is basically the same as the body of the method.

On the other hand, having such methods put together in an utility class might be easier for developers. In the future we might even remove `CustomOrdersTableController::custom_orders_table_usage_is_enabled` (since the class is `Internal`) and leave the functionality in `OrderUtil` alone.

Anyway, if the method is going to stay, I'd rename it as the original one, `custom_orders_table_usage_is_enabled`, since that way its meaning is more clear (with "enabled" alone it isn't clear if it refers to the entire feature or to the table being authoritative).

Reply:
renamed in https://github.com/woocommerce/woocommerce/pull/33638/commits/894953a11604262af023f3574708825513fa0883